llvm/include/llvm-c/CAS/PluginAPI_types.h

```diff
@@ -0,0 +1,85 @@
+/*===-- llvm-c/CAS/PluginAPI_Types.h - CAS Plugin Types Interface -*- C -*-===*\
+|*                                                                            *|
+|* Part of the LLVM Project, under the Apache License v2.0 with LLVM          *|
+|* Exceptions.                                                                *|
+|* See https://llvm.org/LICENSE.txt for license information.                  *|
+|* SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception                    *|
+|*                                                                            *|
+|*===----------------------------------------------------------------------===*|
+|*                                                                            *|
+|* The types for the LLVM CAS plugin API.                                     *|
```

Comment:
How does this API's stability requirements compare to the rest of llvm-c?  I believe llvm-c is only "best effort" ABI stable, so if we need something stronger here it would be worth documenting.

Reply:
I added comment "The API is experimental and subject to change."